index.html

```diff
@@ -8,19 +8,31 @@
 <body>
 <div class="wrap">
    <header>
-        <div class="logo">
+        <nav>
+          <div class="menu-cont">
+            <div class="dst-cont">
+            <div class="logo">
               <a href="#">
                 <img src="images/logo.svg" alt="my logo"/>
               </a>
-        </div>
-         <nav>
+          </div>
+          <div class="dsktp-lf-menu">
+            <ul class="menu-ul">
+              <li><a>Portoflio</a></li>
+              <li><a>About</a></li>
+              <li><a>Contact</a></li>
+            </ul>
+            </div>
+          </div>
              <div class="menu-icon">
                 <img src="images/Union.svg" alt="menu button"/>
```

Comment:
- [ ] Great job on implementing the desktop menu bar, but when you designed it, you missed some issues in the mobile version, `missed the menu icons`, `the desktop menu bar visible in the mobile version`, the `work section not aligning properly`, and also `the about my self section not designed as the template you chose`. Kindly fix :wrench:  the issue addressed in your design as much as possible and stick to your chosen design template.

***Your design***
![Screenshot from 2022-11-11 18-47-15](https://user-images.githubusercontent.com/65129228/201375791-e63b6257-738b-4f00-9ad0-9e8329cf94c7.png)

Reply:
![image](https://user-images.githubusercontent.com/4492335/201453663-e32ffccb-10fa-4e1b-9da9-09a6ec17b760.png)